src/components/SimpleButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+class SimpleButton extends React.Component{
+    render() {
+        return(
+         <Button id="login" variant="outlined" color="primary" size="large" style={{margin:'16px 0px'}} className="Login" onClick={this.props.onClick}>
+            {this.props.label}
+         </Button>
+        )
+    }
+}
```

Comment:
@KalariaYash convert this component to functional as we are not using any state and life cycle method